packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -1931,6 +1939,43 @@ void main() {
     expect(getBorder(tester), disabledBorder);
   });
 
+  testWidgets('OutlineInputBorder radius carries over when lerping', (WidgetTester tester) async {
+    const Key key = Key('textField');
```

Comment:
At the top of the test body, please include:
```
// This is a regression test for https://github.com/flutter/flutter/issues/23982

Reply:
:+1: